dpnp/dpnp_utils/dpnp_utils_statistics.py

```diff
@@ -59,6 +59,11 @@ def _get_2dmin_array(x, dtype):
         It casts to another dtype, if the input array differs from requested one.
 
         """
+        # dtype map for devices without double precision type support
+        dtype_map = {
+            dpnp.float64: dpnp.float32,
+            dpnp.complex128: dpnp.complex64,
+        }
```

Comment:
I'd be better to move the map under the place where `TODO: remove when dpctl.result_type() is fixed` is commented in order to keep in mind that this has to be deleted once the w/a is removing